examples/nlp/neural_machine_translation_with_keras_hub.py

```diff
@@ -46,7 +46,7 @@
 """shell
 pip install -q --upgrade rouge-score
 pip install -q --upgrade keras-hub
-pip install -q --upgrade keras  # Upgrade to Keras 3.
+pip install -q --upgrade keras
```

Comment:
Do we need to install Keras in examples at all now? Colab has Keras 3.x. I think we can remove it?

Reply:
Removed